src/sysinfo.cc

```diff
@@ -222,7 +227,6 @@ void InitializeSystemInfo() {
 // TODO: also figure out cpuinfo_num_cpus
 
 #elif defined OS_WINDOWS
-#pragma comment(lib, "shlwapi.lib")  // for SHGetValue()
```

Comment:
It automatically linked the `shwapi.lib` statically. I think I move it into the build files.